modules/networking/main.tf

```diff
@@ -0,0 +1,25 @@
+resource "google_compute_network" "vpc" {
+  name                    = "${var.namespace}-vpc"
+  description             = "Ctrlplane VPC Network"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "default" {
+  name          = "${var.namespace}-subnet"
+  ip_cidr_range = "10.10.0.0/16"
```

Comment:
This is a huge range just FYI.
https://www.davidc.net/sites/default/subnets/subnets.html?network=10.10.0.0&mask=16&division=1.0

65534 ip's.